solidity/random-beacon/contracts/libraries/Groups.sol

```diff
@@ -164,6 +170,19 @@ library Groups {
         //     self.activeTerminatedGroups.length)
     }
 
+    /// @notice Returns the misbehaved members from the latest group in the
+    ///         group registry.
+    function getLatestMisbehavedMembers(Data storage self)
```

Comment:
If we move it to `DKG.sol` we can have just `dkg.submittedResultMisbehavedMembers`.